src/fetchers/link-fetcher.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+
+import type {PackageRemote, FetchedMetadata, Manifest} from '../types.js';
+import type Config from '../config.js';
+import type {RegistryNames} from '../registries/index.js';
+import * as fs from '../util/fs.js';
+
+export default class LinkFetcher {
```

Comment:
Can you extend from the BaseFetcher so that you only need to override the fetch method?

Reply:
Good call! I'll make that change...